.github/workflows/ci.yml

```diff
@@ -31,4 +31,16 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: npm ci
+
+      # With the way these libraries use each other we need to run our build step for linting in a
```

Comment:
i think it'd be safer to configure stuff like this via lerna, that way we can run a top-level `npm run build` command. i think something like this? https://lerna.js.org/docs/concepts/task-pipeline-configuration#define-task-dependencies-aka-task-pipelines

Reply:
It only seems like we're able to say "run <task> before <task>" but only on a global level. I don't see any way with Lerna/NX to set up something that says "run `oas-normalize` build before `oas` build`.

Reply:
i might play around with turborepo at some point: https://turbo.build/repo/docs/core-concepts/monorepos/task-dependencies